backend/src/test/java/com/woowacourse/levellog/acceptance/InterviewQuestionAcceptanceTest.java

```diff
@@ -0,0 +1,221 @@
+     *   given: 피드백이 등록되어 있다.
+     *   given: 등록된 피드백을 조회한다.
+     *   when: 조회한 피드백을 삭제한다.
+     *   then: 204 No Content 상태 코드를 응답받는다.
+     */
+    @Test
+    @DisplayName("인터뷰 질문 삭제")
+    void deleteById() {
+        // given
+        final RestAssuredResponse hostLoginResponse = login("페퍼");
+        final Long pepperId = hostLoginResponse.getMemberId();
+        final String pepperToken = hostLoginResponse.getToken();
+
+        final RestAssuredResponse romaLoginResponse = login("로마");
+        final Long romaId = romaLoginResponse.getMemberId();
+        final String romaToken = romaLoginResponse.getToken();
+
+        final TeamCreateDto teamCreateDto = new TeamCreateDto("롬펲 인터뷰", "트랙룸", LocalDateTime.now().plusDays(3),
+                new ParticipantIdsDto(List.of(pepperId, romaId)));
+        final String teamId = post("/api/teams", pepperToken, teamCreateDto).getTeamId();
+
+        final LevellogDto levellogRequest = LevellogDto.from("페퍼의 레벨로그");
+        final String pepperLevellogId = post("/api/teams/" + teamId + "/levellogs", pepperToken,
+                levellogRequest).getLevellogId();
+
+        final String interviewQuestionId = requestSaveInterviewQuestion(pepperLevellogId, romaToken,
+                "Spring을 사용하는 이유?").getInterviewQuestionId();
+
+        // when
+        final ValidatableResponse response = RestAssured.given(specification).log().all()
+                .header(HttpHeaders.AUTHORIZATION, "Bearer " + romaToken)
+                .contentType(MediaType.APPLICATION_JSON_VALUE)
+                .filter(document("interview-question/delete"))
+                .when()
+                .delete("/api/levellogs/{levellogId}/interview-questions/{interviewQuestionId}",
+                        pepperLevellogId, interviewQuestionId)
+                .then().log().all();
+
+        // then
+        response.statusCode(HttpStatus.NO_CONTENT.value());
+        requestFindAllInterviewQuestion(pepperLevellogId, romaToken)
+                .body("interviewQuestions.id", not(contains(interviewQuestionId)));
+    }
+
+    private RestAssuredResponse requestSaveInterviewQuestion(final String levellogId,
+                                                             final String fromMemberToken,
+                                                             final String interviewQuestion) {
+        return post("/api/levellogs/" + levellogId + "/interview-questions", fromMemberToken,
+                InterviewQuestionDto.from(interviewQuestion));
+    }
+
+    private ValidatableResponse requestFindAllInterviewQuestion(final String levellogId,
+                                                                final String fromMemberToken) {
+        return RestAssured.given().log().all()
+                .header(HttpHeaders.AUTHORIZATION, "Bearer " + fromMemberToken)
+                .when()
+                .get("/api/levellogs/" + levellogId + "/interview-questions")
+                .then().log().all();
```

Comment:
```suggestion
        return get("/api/levellogs/" + levellogId + "/interview-questions", fromMemberToken).getResponse();
```
`RestAssuredTemplate.get` 을 활용해보는 건 어떨까요?